web/src/pages/OGTagPreviewPage/PreviewButton.tsx

```diff
@@ -0,0 +1,81 @@
+import { useLazyQuery } from '@apollo/client'
+import { Button } from '@tremor/react'
+import { OGTagPreviewProviderAudit, OGTagPreviewResponse } from 'types/graphql'
+
+import { RefreshIcon } from 'src/icons/Icons'
+
+const OG_TAG_PREVIEW_QUERY = gql`
```

Comment:
Could/should this also be a Cell?

Reply:
I don't think it has to be.

Reply:
Maybe because of the way you pass in `setError` etc it's different 🤔